### DISCUSSION TITLE
Backup /var/lib/docker/volumes/nginx_certs/ ?

### DISCUSSION THREAD

**Discussion by dm17** [nginx-proxy/acme-companion#805] [General]
Do we need to safeguard the host's volumes? I imagine a backup of nginx_certs is necessary - and we just copy them back on a new system, for example? Otherwise it seems none of the docker-letsencrypt-nginx-proxy-companion has backup concerns on production, right?
Thanks.

**Comment by buchdag** [comment]
> Otherwise it seems none of the docker-letsencrypt-nginx-proxy-companion has backup concerns on production, right?

Right, there is no backup mechanism built inside the container, that's left to the user.

You can copy the whole `certs` folder using `docker cp`. Let say you letsencrypt container name is `le-companion`:

```
docker cp -a le-companion:/etc/nginx/certs "$(pwd)/certs"
```
